client/src/components/layouts/global.css

```diff
@@ -28,7 +29,7 @@ body {
 
 .codeally-frame {
   display: block;
-  height: calc(100vh - var(--header-height, 0px));
```

Comment:
I have added different fallback, because

- this fallback was 0 if `--header-height` doesn't exist, but that isn't necessary because if `header-height` isn't exist then it's 0, so there is no need to fallback.
- instead of removing it, I have added the values as fallback, because this can act as fallback if something is messed up.

Reply:
Do we want to add fallbacks to our variables? I'm fine with it, but it feels like it defeats the purpose of variables. If we ever change the header height, then we have to change all the fallbacks.

Reply:
Makes sense 👍